torch/nn/functional.py

```diff
@@ -2520,6 +2520,8 @@ def group_norm(
     """
     if has_torch_function_variadic(input, weight, bias):
         return handle_torch_function(group_norm, (input, weight, bias,), input, num_groups, weight=weight, bias=bias, eps=eps)
+    if input.dim() < 2:
```

Comment:
What would happen if `input.dim()` was 0 or 1 previously?

Reply:
It would fail at the next line.
 `_verify_batch_size([input.size(0) * input.size(1) // num_groups, num_groups] + list(input.size()[2:]))`
`IndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)`